mmv1/products/certificatemanager/api.yaml

```diff
@@ -253,20 +292,6 @@ objects:
         name: 'description'
         description: |
           One or more paragraphs of text description of a certificate map entry.
-      - !ruby/object:Api::Type::String
-        name: 'createTime'
-        output: true
-        description: |
-          Creation timestamp of a Certificate Map. Timestamp is in RFC3339 UTC "Zulu" format,
-          accurate to nanoseconds with up to nine fractional digits.
-          Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
-      - !ruby/object:Api::Type::Time
-        name: 'updateTime'
-        description: |
-          Update timestamp of a Certificate Map. Timestamp is in RFC3339 UTC "Zulu" format,
-          accurate to nanoseconds with up to nine fractional digits.
-          Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
-        output: true
```

Comment:
Would you mind if we keep the already implemented `createTime` and `updateTime` fields in the providers. Honestly removing those fields is likely not going to affect any users but removing a field is considered a breaking change in general which we can not release in minor release. Just want to be extra cautious and don't want to break any existing users.